_data/tumbleweed.yml

```diff
@@ -117,56 +117,68 @@ downloads:
         url: /ports/zsystems/tumbleweed/iso/openSUSE-Tumbleweed-NET-s390x-Current.iso?mirrorlist
       - name: checksum
         url: /ports/zsystems/tumbleweed/iso/openSUSE-Tumbleweed-NET-s390x-Current.iso.sha256
-- name: jeos_images
+- name: minimal-vm_images
```

Comment:
parser complains about this line
/home/lkocman/Workspace/opensuse/get-o-o/vendor/bundle/ruby/2.7.0/gems/safe_yaml-1.0.5/lib/safe_yaml/load.rb:143:in `parse': (/home/lkocman/Workspace/opensuse/get-o-o/_data/tumbleweed.yml): did not find expected key while parsing a block mapping at line 120 column 7 (Psych::SyntaxError)

Reply:
oops, should be fixed now